tests/application/containers/php/php.Dockerfile

```
@@ -1,31 +1,51 @@
+    && adduser --uid=$UID --gid=$GID --shell=/bin/bash --comment "" --disabled-password php
 
-FROM base AS dependencies
+USER $UID:$GID
+
+WORKDIR /home/php/app
+
+FROM php-base AS php-base-with-composer
+
+USER root
+
+RUN \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)